### PR TITLE
Fix info messages

### PR DESCRIPTION
**Pull request by andriylesch**

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = class extends yeoman {
   }
 
   configuring() {
-    this.log('configuring');
+    // This.log('configuring');
```
Comment (peppelan) [inline generators/app/index.js]: this change does not seem intentional...

File: generators/app/index.js [hunk 2]
```diff
@@ -341,7 +341,7 @@ module.exports = class extends yeoman {
       '**********************************************************************************'
     );
     this.log(
-      '* Your project is now created, you can use the following commands to get going   *'
+      '* Your project is now created, follow instruction bellow                         *'
```
Comment (peppelan) [inline generators/app/index.js]: "follow *the* instruction*s* be*l*ow"

File: generators/pkgg/index.js [hunk 3]
```diff
@@ -164,7 +164,7 @@ module.exports = class extends yeoman {
       '**********************************************************************************'
     );
     this.log(
-      '* Your package is now created, you can follow description and start using *'
+      '* Your package is now created, follow instruction bellow                         *'
```
Comment (peppelan) [inline generators/pkgg/index.js]: "follow *the* instruction*s* be*l*ow"

File: generators/app/index.js [hunk 4]
```diff
@@ -41,9 +41,9 @@ module.exports = class extends yeoman {
 
     if (!this.IsContinue) {
       this.log(
-        `Create progect in directory '${chalk.red(
+        `YoGo generator will only generate project in '${chalk.red(
           'GOPATH/src/<YOUR_PROJECT>'
-        )}'. YoGo generator will stop execution.`
+        )} directory'. Otherwise process will be stopped.`
```
Comment (peppelan) [inline generators/app/index.js]: *the* process ?

Comment (andriylesch) [reply generators/app/index.js]: maybe better - 'generation'

File: generators/app/index.js (hunk 1 above)
Comment (andriylesch) [reply generators/app/index.js]: removed

File: generators/app/index.js (hunk 2 above)
Comment (andriylesch) [reply generators/app/index.js]: done

File: generators/pkgg/index.js (hunk 3 above)
Comment (andriylesch) [reply generators/pkgg/index.js]: done

**Review by peppelan — APPROVED** [review]

**Pull request merged**